qcodes/tests/conftest.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+import qcodes as qc
+
+
+@pytest.fixture(scope="session", autouse=True)
+def disable_telemetry():
+    """
+    We do not want the tests to send up telemetric information. This fixture
+    can be imported to disable the telemetry. Each `test_*.py` file that
```

Comment:
Last part of the docstring is out of date?

Reply:
Good point!